indico/modules/events/registration/controllers/api/misc.py

```diff
@@ -5,10 +5,19 @@
 # modify it under the terms of the MIT License; see the
 # LICENSE file for more details.
 
+from itertools import groupby
+
 from indico.modules.events.controllers.base import RHProtectedEventBase
 
 
 class RHAPIRegistrationForms(RHProtectedEventBase):
     def _process(self):
         from indico.modules.events.registration.schemas import RegistrationFormPrincipalSchema
         return RegistrationFormPrincipalSchema(many=True).jsonify(self.event.registration_forms)
+
+
+class RHAPIEventTimeTable(RHProtectedEventBase):
```

Comment:
something like `RHAPISessionBlocks` would be more descriptive of what this code here does

Reply:
What about `RHAPIEventSessionBlocks` adding event to describe you're fetching an event

Reply:
yeah, fine for me as well